L/libcxxwrap_julia/build_tarballs.jl

```diff
@@ -11,7 +11,7 @@ delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
 include("../../L/libjulia/common.jl")
 
 name = "libcxxwrap_julia"
-version = v"0.13.3"
+version = v"0.13.4"
```

Comment:
I now bumped the version here. However, this introduces a mismatch between the libcxxwrap version and the jll one.

I would leave it up to @barche to decide on how to deal with this.

Reply:
yes. and in the meantime we can see whether it works for GAP_jll

Reply:
I will make a release for v0.13.4, identical to v0.13.3 except for the version number, so this PR will need to be updated with the new SHA once this is done.

Reply:
Updated now, should be good to go after the rebuild.

Reply:
Thanks @barche! Looks good to me